.github/workflows/build-and-push-to-docker.yml

```diff
@@ -698,3 +702,223 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           cache-from: type=gha
           provenance: false
+
+  build-vscode-base:
+    needs: lint
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: true
+      matrix:
+        include:
+          - DEBIAN_VERSION: debian
+          - DEBIAN_VERSION: bookworm
```

Comment:
Because I suspect in the future we will add more versions. For now they are the same.

Reply:
sorry to be annoying here and feel free to ignore if you disagree, but since there's just one and bookworm is a designation of a specific debian, would it make sense to just have `debian`? Or are you worried that at some point it will be switched?

Reply:
I thought at some point we will use a different debian version. I will just put the base image directly here - rather than `DEBIAN_VERSION`. So it will just be `BASE_IMAGE` and we can set that to whatever. Although only images based on debian will work. But if we decide to go use ubuntu it will be easier.

Reply:
And yeah now the image tags will just say `debian`.